ir/README.md

```diff
@@ -28,6 +28,6 @@ Although generation of an `AirIR` uses a symbol table while processing the sourc
 - **Public inputs**, represented by a vector that maps an identifier to a size for each public input that was declared. (Currently, public inputs can only be declared as fixed-size arrays.)
```

Comment:
nit: is the sentence "...to a size..." complete?

Reply:
@Al-Kindi-0 It seems complete to me. Not sure if I'm missing something.

Reply:
Technically it's not complete, because we have a comma instead of a verb (instead of "Public inputs are represented by a vector that maps..."). That's because it's really a continuation of the sentence above the set of bullet points. It might be clearer to say "Public inputs, which are represented by...", in which case the other bullet points should be similarly updated.

Reply:
But it's technically correct as it is, as long as the whole chunk is read together (it's just not a sentence on its own)